scripts/provision-elasticsearch.sh

```diff
@@ -140,30 +148,36 @@ main () {
         os_distribution=$(cut -d ' ' -f 1 /etc/redhat-release)
         os_release=$(grep -oE '[0-9]+\.' /etc/redhat-release | cut -d. -f1 | head -1)
     else
-        os_distribution="UNDETECTED"
-        os_release="UNDETECTED"
+        os_distribution="${1:-UNDETECTED}"
```

Comment:
This is a bit weak, people can accidentally pass unwanted arguments.  Elsewhere, we test for Docker presence like:

```
   if [ -f /.dockerinit -o -f /.dockerenv ]; then
        # running inside Docker
```

It would be good to do the same philosophy everywhere.
```